pkg/validations/proxy_validation_test.go

```diff
@@ -52,6 +53,28 @@ func Test_TestProxyUrlConnection(t *testing.T) {
 	}
 }
 
+func Test_ValidateProxyParameters(t *testing.T) {
+	config := commonproto.ProxyConfiguration{}
+	proxy := NewProxy()
+	defer proxy.Target.Close()
+	config.HttpProxy = proxy.Target.URL
+	config.HttpsProxy = proxy.Target.URL
+
+	// valid case
+	err := ValidateProxyParameters(&config)
+	if err != nil {
+		t.Fatalf("Test_ValidateProxyParameters test case failed. %s", err.Error())
+	}
+
+	// invalid case - invalid url
+	config.HttpProxy = "//akse2e:akse2e@skyproxy.ceccloud1.selfhost.corp.microsoft.com:3128"
```

Comment:
hope this doesnt cause a credential scanner to raise a bug

Reply:
Is this the only proxy server format we are planning to use?

Reply:
I removed credentials